docs/src/main/sphinx/release/release-408.md

```diff
@@ -0,0 +1,55 @@
+# Release 408 (22 Feb 2023)
+
+## General
+
+* Fix query failure when performing a remote partitioning exchange on a
+  [structural type](structural-data-types) that contains `NULL` elements. ({issue}`16140`)
+
+## CLI
+
+* Add `AUTO` output format which switches from `ALIGNED` to `VERTICAL` if
+  the output doesn't fit the current terminal. ({issue}`12208`)
+## BigQuery connector
+
+* Add support for writing BigQuery `numeric` and `bignumeric` types. ({issue}`16145`)
+
+## Delta Lake connector
+
+* Add support for creating tables with the `change_data_feed_enabled` table
+  property. ({issue}`16129`)
+* Improve query performance on tables that Trino has written to with `INSERT`. ({issue}`16026`)
+* Retry dropping Delta tables registered on AWS Glue to avoid failures due to
+  concurrent modifications. ({issue}`13199`)
+* Fix inaccurate change data feed entries for `MERGE` queries. ({issue}`16127`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Hive connector
+
+* Retry dropping Delta tables registered on AWS Glue to avoid failures due to
+  concurrent modifications. ({issue}`13199`)
+* Remove support for the deprecated `hive-hadoop2` connector name. ({issue}`16166`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Hudi connector
+
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## Iceberg connector
+
+* Reduce memory usage when reading `$files` system tables. ({issue}`15991`)
+* Require the `iceberg.jdbc-catalog.driver-class` configuration property to be
+  set to prevent a "driver not found" error after initialization. ({issue}`16196`)
+* Fix performance regression for writing to partitioned tables when table
+  statistics are absent. ({issue}`16152`)
+
+## SingleStore connector
+
+* Remove support for the deprecated `memsql` connector name. ({issue}`16180`)
+
+## SQL Server connector
+
+* Add support for predicate pushdown on case-sensitive `varchar` and
+  `nvarchar` columns with the `IN`, `NOT IN`, `==`, and `<>` operators. ({issue}`15714`)
```

Comment:
```suggestion
* Add support for pushing down `IN`, `NOT IN`, `=` and `!=` predicates on text columns which use a case-sensitive collation in SQL Server. ({issue}`15714`)
```

(There's no `==`, also it's not limited to varchar and nvarchar, it applies to any text types)